src/test/java/seedu/task/commons/util/UrlUtilTest.java

```diff
@@ -1,14 +1,12 @@
 package seedu.task.commons.util;
 
```

Comment:
Even if it increases the test coverage, it's not right. In fact, we might even be penalised for not morphing address book properly and keeping unused stuff There is no reason for this to be here anyway. We should delete the BrowserPanel and the other related tests